dev/lint.sh

```diff
@@ -6,7 +6,12 @@ err=0
 trap 'err=1' ERR
 
 echo -e "\n========== black ==========\n"
-black --check . 2>&1 | sed 's/^would reformat \(.*\)/\1: This file is unformatted. Run `black .` or comment `autoformat` on the PR to format./'
+if [ -z "${GITHUB_ACTIONS}" ]; then
+  black --check .
+else
+  black --check . > .black-output 2>&1
+  sed 's/^would reformat \(.*\)/\1: This file is unformatted. Run `black .` or comment `autoformat` on the PR to format./' .black-output
+fi
```

Comment:
On local, just run `black --check .`.
On GitHub Actions, apply `sed` for problem matchers.